.github/workflows/main.yml

```diff
@@ -94,6 +94,20 @@ jobs:
       run: |
         bash <(curl -s https://codecov.io/bash)
 
+  check_build_reproducibility:
+    name: 'Check build reproducibility'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: 'Set up JDK 11'
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Build and compare checksums
+      shell: bash
+      run: |
+        ./src/checkBuildReproducibility.sh
+
```

Comment:
Let's move this to a dedicated workflow so it can run in parallel with the main workflow.